discovery_engine_api_tests/handlers/api_handler.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+import requests
+import allure
+import logging
+from config.config import Config
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ApiHandler:
+    TIMEOUT = 5
+
+    def __init__(self):
+        conf = Config()
+        self.interactions = conf.get_interactions_endpoint()
+        self.ingestion = conf.get_ingestion_endpoint()
```

Comment:
:art:  Maybe the name should indicate that it's an endpoint or uri. As it isn't a container of interactions. So if you jump to some test and read `self.interactions.format` it can be a bit confusing.